content/projekte/familienlabore.en.md

```diff
@@ -20,7 +20,7 @@ contact:
   website: https://familienlabore.de
 
 ---
-The shaping of family life in Germany has changed enormously in recent decades. Patchwork families, multi-generation houses and households of single parents bring completely different requirements with them. One of the biggest challenges is the compatibility of family and career. Or to put it another way: Is it really necessary to have to sit in a doctor's waiting room for three hours each time? We think: this works better.
+The shaping of family life in Germany has changed enormously in recent decades. Patchwork families, multi-generation houses and households of single parents bring completely different requirements with them. One of the biggest challenges is the compatibility of family and career. Or to put it another way: is it really necessary to have to sit in a doctor's waiting room for three hours each time? We think that this can work better.
```

Comment:
This is really confusing. What does the compatibility of family and career have to do with sitting in a doctor's waiting room?

Reply:
I think this is inspired by something like https://www.moers.de/app/inhalt/wartezeiten-buergerservice/